### PR TITLE
1664 Upgrade /desktop/ubuntu-kylin to Vanilla v1

### LINKED ISSUE #1664: /desktop/ubuntu-kylin/ to VBT

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

 Upgrade /desktop/ubuntu-kylin to Vanilla v1

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/desktop/ubuntu-kylin/](http://0.0.0.0:8001/desktop/ubuntu-kylin/))
- Ensure updated pages matches [patterns](https://www.dropbox.com/home/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/5.%20Desktop?preview=desktop-kylin.png) as outlined

## Issue / Card

Fixes #1664 

## Demo

http://www.ubuntu.com-1664-desktop-kylin.demo.haus

### CODE REVIEW

**Comment by yaili** [comment]
Design review:
- [ ] The hero is missing the image
- [ ] First link in hero should be a brand button
- [ ] Image in "A global partnership" should be centred horizontally in the available space
- [ ] Add border above and below "What’s new in Ubuntu Kylin 16.04 LTS?"
- [ ] Add border above and below "China’s own Ubuntu"
- [ ] Add border above and below "Youker Weather"
- [ ] Further reading in contextual footer should have no bullets
- [ ] In S screen, the image in "A global partnership" should be centred

**Comment by barrymcgee** [comment]
@yaili Fixes actioned, demo updated.

**Comment by yaili** [comment]
- [ ] China should be capitalised in the breadcrumb and the nav dropdown

**Comment by barrymcgee** [comment]
@yaili Good catch.. fixed & updated demo.

File: templates/desktop/ubuntu-kylin.html [hunk 1]
```diff
@@ -1,187 +1,188 @@
...
- </section>
-
-{% include "shared/contextual_footers/_contextual_footer.html"  with first_item="_desktop_kylin_download" second_item="_desktop_kylin_learn_more" third_item="_desktop_further_reading" %}
+<div class="p-strip is-bordered">
+  <div class="row">
+    <div class="u-equal-height">
+      <div class="col-6">
+      <h1>Ubuntu Kylin for&nbsp;China</h1>
+      <p>Ubuntu Kylin is an official flavour of Ubuntu. It is a free PC operating system created for China and complies with the Chinese government procurement regulations. It includes all the features you&rsquo;ve come to expect from Ubuntu, alongside essential Chinese software and apps. The interface has been designed specifically to put Chinese users first &mdash; and with support for touch screens and HiDPI monitors, it runs beautifully on all kinds of hardware.</p>
+      <p><a href="http://cn.ubuntu.com" class="p-button--brand">Ubuntu 中国网站现已面世</a></p>
```
Comment (pmahnke) [inline templates/desktop/ubuntu-kylin.html]: should this have &lt;span class="p-link--external">?

File: templates/desktop/ubuntu-kylin.html [hunk 2]
```diff
@@ -1,187 +1,188 @@
...
+      <h3>Kingsoft WPS Office</h3>
+      <p>The most popular office suite in China is available on Ubuntu Kylin. We&rsquo;ve worked closely with Kingsoft, so you can make full use of its WPS Office package for word processing, spreadsheets and presentations.</p>
+    </div>
+    <div class="col-6 u-align--center">
+      <img src="{{ ASSET_SERVER_URL }}43b1abca-desktop-kylin-kingsoft.jpg" alt="" width="488" height="260" />
+    </div>
+  </div>
+</div>
+
+{% include "shared-v1/contextual_footers/_contextual_footer.html"  with first_item="_desktop_kylin_download" second_item="_desktop_kylin_learn_more" third_item="_desktop_further_reading" %}
```
Comment (pmahnke) [inline templates/desktop/ubuntu-kylin.html]: should the _desktop_kylin_learn_more use an external link too?

**Pull request merged**